docs/UML/LoadEvents.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+hide footbox
+hide circle
+
+-> ":Storage" : loadEvents()
+activate ":Storage"
+":Storage" -> ":EventListStorage" : loadEvents()
+activate ":EventListStorage"
+":EventListStorage" ->":EventListAdapter" : gson.fromJson(fileReader, ArrayList.class)
```

Comment:
Is this correct? I was under the assumption that when your arrow is drawn to a class, you are calling a method in that class. Is `fromJson()` a method in `:EventListAdapter`?